clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -716,23 +716,21 @@ public void testInterceptorConstructorClose(GroupProtocol groupProtocol) {
     @ParameterizedTest
     @EnumSource(GroupProtocol.class)
     public void testInterceptorConstructorConfigurationWithExceptionShouldCloseRemainingInstances(GroupProtocol groupProtocol) {
-        final int targetInterceptor = 3;
+        final int targetInterceptor = 1;
```

Comment:
The test doesn't quite match the name of the test since there are no remaining instances. It seems that it's still useful to verify if remaining instances are closed?

Reply:
I’ve updated the test name to `testInterceptorConstructorExceptionShouldCloseCreatedInstance`